src/Microsoft.DotNet.ProjectJsonMigration/ProjectMigrator.cs

```diff
@@ -99,6 +97,11 @@ private void VerifyProject(IEnumerable<ProjectContext> projectContexts, string p
             }
         }
 
+        private string FormatDiagnosticMessage(DiagnosticMessage d)
```

Comment:
The formatting is non-ideal, the file path up front makes it difficult to read the error.